src/test/debuginfo/vec-slices.rs

```diff
@@ -53,6 +53,11 @@
 // gdb-command:print *((int64_t[2]*)('vec_slices::MUT_VECT_SLICE'.data_ptr))
 // gdb-check:$15 = {64, 65}
 
+//gdb-command:print mut_slice.length
```

Comment:
Actually, these tests are unnecessary,  we just need a test that verifies that printing mut_slice gives properly formatted results.